plugins/jobs/src/js/pages/JobConfiguration.js

```diff
@@ -0,0 +1,139 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import ConfigurationMap from "#SRC/js/components/ConfigurationMap";
+import ConfigurationMapHeading from "#SRC/js/components/ConfigurationMapHeading";
+import ConfigurationMapLabel from "#SRC/js/components/ConfigurationMapLabel";
+import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
+import ConfigurationMapSection from "#SRC/js/components/ConfigurationMapSection";
+import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
+import HashMapDisplay from "#SRC/js/components/HashMapDisplay";
+
+export default class JobConfiguration extends React.Component {
+  static getSchedule(job) {
```

Comment:
❓Is this legacy code? Also, where is this used? I'm asking because this should be handled by `job.schedules.something`…

Reply:
Yes, I copied too much there